polkadot/scripts/build-only-wasm.sh

```diff
@@ -22,8 +22,8 @@ fi
 if [ -d $WASM_BUILDER_RUNNER ]; then
   export DEBUG=false
   export OUT_DIR="$PROJECT_ROOT/target/release/build"
-  cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" \
+  forklift cargo run --release --manifest-path="$WASM_BUILDER_RUNNER/Cargo.toml" \
     | grep -vE "cargo:rerun-if-|Executing build command"
 else
-  cargo build --release -p $1
+  forklift cargo build --release -p $1
```

Comment:
Actually i think this script is meant for endusers - not for CI.  
Can you maybe run it through the `RUSTC_WRAPPER` env variable like `RUSTC_WRAPPER=forklift ./build-only-wasm.sh`?

Reply:
Makes sense. But wrappers need a "coordinator"  #process, and I don't like the idea of copy/paste script just for ci, so I added check for forklift

Reply:
Yup sounds good.